src/config.cc

```diff
@@ -1157,8 +1157,10 @@ Class load()
 
   c.showingDictBarNames = ( root.namedItem( "showingDictBarNames" ).toElement().text() == "1" );
 
-  c.usingToolbarsIconSize =
-    ( static_cast< ToolbarsIconSize >( root.namedItem( "usingToolbarsIconSize" ).toElement().text().toInt() ) );
+  QDomNode usingToolbarsIconSize = root.namedItem( "usingToolbarsIconSize" );
+  if ( !usingToolbarsIconSize.isNull() ) {
```

Comment:
when the usingToolbarsIconSize = null ,there should be a default value?

Reply:
The default is normal.

https://github.com/xiaoyifang/goldendict-ng/blob/96ada0737cbde5f7cefdd24262f0d04b000055df/src/config.hh#L878